ramhorns/src/traits.rs

```diff
@@ -34,120 +40,124 @@ pub trait ContentSequence: Combine + Sized + Copy {
     /// Render a field by the hash **or** string of its name.
     ///
     /// This will escape HTML characters, eg: `<` will become `&lt;`.
-    /// If successful, returns `true` if the field exists in this content, otherwise `false`.
     #[inline]
     fn render_field_escaped<E: Encoder>(
         &self,
         _hash: u64,
         _name: &str,
         _encoder: &mut E,
-    ) -> Result<bool, E::Error> {
-        Ok(false)
+    ) -> Result<(), E::Error> {
```

Comment:
Sorry for reviewing so late. I'm using the return values in the `#[ramhorns(flatten)]` IIRC, so I'm not sure if this breaks things. I'm actually surprised CI passed here.

Reply:
As far as I see, only the return values of the `Content` trait (not `ContentSequence` trait) are used in the `flatten` implementation. I can bring these back if a future use is planned, though.
I just changed the return types because these values were not used anywhere at the moment.

Reply:
Ah, that makes more sense, I should have read more carefully!